easybuild/easyconfigs/b/bmtagger/bmtagger-3.101_fix-hardcoding.patch

```diff
@@ -0,0 +1,11 @@
+fix hardcoding of compiler command & flags, by passing down $CXX, $CXX and $CXXFLAGS
```

Comment:
you're passing down CFLAGS, not CXXFLAGS?

Reply:
@wpoely86 The code is partially C & C++, and the flags are identical anyway...